pkg/controller/admissionchecks/multikueue/controllers.go

```diff
@@ -20,15 +20,20 @@ import (
 	"time"
 
 	ctrl "sigs.k8s.io/controller-runtime"
+
+	kueuealpha "sigs.k8s.io/kueue/apis/kueue/v1alpha1"
+	"sigs.k8s.io/kueue/pkg/controller/jobframework"
 )
 
 const (
-	defaultGCInterval        = time.Minute
-	defaultOrigin            = "multikueue"
-	defaultWorkerLostTimeout = 5 * time.Minute
+	defaultGCInterval             = time.Minute
+	defaultOrigin                 = "multikueue"
+	defaultWorkerLostTimeout      = 5 * time.Minute
+	defaultMultikueControllerName = kueuealpha.MultiKueueControllerName
```

Comment:
There is no need for this, just use `kueuealpha.MultiKueueControllerName` when setting the defaults.